src/TulsiGenerator/Scripts/bazel_build.py

```diff
@@ -1556,6 +1563,17 @@ def _ExtractTargetSourcePaths(self):
 
     return source_path_prefixes
 
+  def _LinkTulsiWorkspace(self):
+    """Links the Bazel Workspace to the Tulsi Workspace ( `tulsi-workspace` )"""
+    bazel_workspace = self.bazel_symlink_prefix + os.path.basename(self.workspace_root)
```

Comment:
The source path you are looking for is already resolved in `self.bazel_build_workspace_root`